tutorials/cpp/bidi.cpp

```diff
@@ -9,8 +9,8 @@
 #include <iostream>
 
 int main() {
-    ICU4XDataProvider dp = ICU4XDataProvider::create_compiled();
-    auto bidi = ICU4XBidi::create(dp).ok().value();
+    std::unique_ptr<ICU4XDataProvider> dp = ICU4XDataProvider::create_compiled();
+    std::unique_ptr<ICU4XBidi> bidi = ICU4XBidi::create(*dp.get()).ok().value();
```

Comment:
the whole `*<_>.get()` situation to use a `std::unique_ptr` in a reference location is annoying.